omero_marshal/decode/decoders/channel.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .. import Decoder
+from omero.model import ChannelI, LogicalChannelI
+
+
+class Channel201501Decoder(Decoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Channel'
+
+    OMERO_CLASS = ChannelI
+
+    def decode(self, data):
+        v = super(Channel201501Decoder, self).decode(data)
+        self.set_property(v, 'alpha', data.get('Alpha'))
+        self.set_property(v, 'blue', data.get('Blue'))
+        self.set_property(v, 'green', data.get('Green'))
+        self.set_property(v, 'red', data.get('Red'))
+        self.set_property(v, 'lookupTable', data.get('omero:lookupTable'))
+
+        logical_channel_id = data.get('omero:LogicalChannelId')
+        if logical_channel_id is None:
+            return v
```

Comment:
If I'm creating a new Channel, to save to OMERO, I won't have a LogicalChannelId. So how do I create a new LogicalChannel in OMERO? Can we not create a new, unsaved LogicalChannel here?